docs/index.asciidoc

```diff
@@ -24,6 +24,11 @@ The CSV filter takes an event field containing CSV data, parses it,
 and stores it as individual fields with optionally-specified field names.
 This filter can parse data with any separator, not just commas.
 
+[id="plugins-{type}s-{plugin}-ecs_metadata"]
+==== Event Metadata and the Elastic Common Schema (ECS)
+When ECS compatibility is disabled, the value is stored in the root level.
+When ECS is enabled, the value is stored in the `target` field.
+
```

Comment:
I haven't added the table to compare the columns of ECS disabled and ECS v1 because the field name is depending on the value of `target` which has no default value

Reply:
I understand the intention but from a plugin user perspective the docs seems confusing.
Maybe we should just include the same message as the one that gets logged to explain?

```
The plugin behaves the same regardless of ECS compatibility, except issuing a warning when `target` isn't set.
It is recommended to set the `target` option to avoid potential schema conflicts.
```
(just a draft - probably need rewording if we decide to use it)